src/general/smoothing_kernels.jl

```diff
@@ -103,7 +103,7 @@ end
 @doc raw"""
     SchoenbergCubicSplineKernel{NDIMS}()
 
-Cubic spline kernel by Schoenberg (Schoenberg, 1946), given by
+Cubic spline kernel by [Schoenberg1946](@cite), given by
```

Comment:
Why do you cite like this here but in text everywhere before?
Same for the other smoothing kernels.